test/run-drun/ok/actor-class-mgmt.drun-run.ok

```diff
@@ -1,3 +1,3 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
-ingress Completed: Reply: 0x4449444c0000
+ingress Completed: Reject: IC0302: Management canister has no method 'canister_info'
```

Comment:
Our `drun` doesn't support this call yet. #3992 is having problems. But we'll be able to test this in the Playground.